app/src/main/java/com/android/wardrobeManager/ui/images/filters/DiagonalDownRight.java

```diff
@@ -0,0 +1,29 @@
+package com.android.wardrobeManager.ui.images.filters;
+
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.PorterDuff;
+import android.graphics.PorterDuffXfermode;
+
+import com.android.wardrobeManager.ui.images.DesignFilterManager;
+
+import java.util.Map;
+
+public class DiagonalDownRight implements DesignFilterManager.DesignFilter {
+    public Bitmap filter(Bitmap bitmap, Bitmap ref, int[] colors) {
+        int colorIndex = 0;
+        int thickness = 30;
+        Canvas canvas = new Canvas(bitmap);
+        Paint paint = new Paint(Paint.ANTI_ALIAS_FLAG);
+        paint.setXfermode(new PorterDuffXfermode(PorterDuff.Mode.SRC_IN));
+        paint.setStrokeWidth(thickness);
+
+        for (int i = -bitmap.getHeight(); i < bitmap.getHeight(); i += thickness) {
+            paint.setColor(colors[colorIndex]);
+            canvas.drawLine(0, i, bitmap.getWidth(), i + bitmap.getHeight(), paint);
```

Comment:
Using drawLine is pretty inefficient. It would be better to draw rotated rectangles. If you do it this way, then you can pass the angle into the constructor, then you only need one class as well.